tests/git-ftp-test.sh

```diff
@@ -151,6 +151,24 @@ test_init_fails() {
 	#assertEquals 1 $error_count
 }
 
+test_init_invalid_password() {
+	REMOTE_BASE_URL_DISPLAY="ftp://$GIT_FTP_USER:***@$GIT_FTP_HOST$GIT_FTP_PORT"
+	cd $GIT_PROJECT_PATH
```

Comment:
This is already done in the setUp method. It's still in a lot of test methods from a time when it was not done in setUp. But we can actually remove them all (#spring-clean).